dev/kubernetes/k3d-dev-env-setup.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+mkdir ~/.kube
+
+
+k3d registry \
+    create mlflow-registry.localhost \
+    --port 12000
```

Comment:
If we run this command twice, would the second run fail?

Reply:
Fixed, see above comment re adding `set -ex`